testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1,9 +1,16 @@
 # Copyright (c) 2010-2023 SUSE LLC.
 # Licensed under the terms of the MIT license.
 
+### This file contains all step definitions concerning
```

Comment:
I am wondering why you are breaking the line here already. When looking at the other comments you did, the line length is much longer then e.g. this one.

Reply:
I don't have an answer to that. I will reformat it to be a little bit longer 👍🏼